site/en/blog/screen-sharing-improvements-in-chrome-109/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Screen sharing improvements in Chrome 109'
+description: >
+ Details of two screen sharing improvements for the web—a mechanism for controlling tab focus when screen sharing, and a mechanism to control local audio playback.
+authors:
+  - beaufortfrancois
+  - eladalon
+date: 2022-11-16
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/1gYV3BbeqdYWvolyzP0E.jpg
+alt: Brown wooden letter t-embossed decor photo.
+tags:
+  - chrome-109
+---
+
+Web apps can already use [`getDisplayMedia()`] to capture tabs, windows or screens as a [MediaStream]. From Chrome&nbsp;109, you can take advantage of the following improvements:
+
+- When screen sharing starts, [Conditional Focus] allows the capturing web app to control whether the browser activates the captured tab or window, or whether the capturing tab remains active.
```

Comment:
This is the only place we use "activates" instead of "focuses". It might be better to change to always using "focuses". Wdyt?

Reply:
Done. Note that it was part of our original doc.

Reply:
Noted.